de.akaflieg_freiburg.enroute.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "de.akaflieg_freiburg.enroute",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "enroute",
+    "finish-args": [
+	"--filesystem=host",
+        "--share=ipc",
+	"--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+	"--system-talk-name=org.freedesktop.GeoClue2",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "enroute",
+            "buildsystem": "cmake-ninja",
+            "config-opts": ["-DCMAKE_BUILD_TYPE=RelWithDebInfo"],
+            "sources": [
+                {
+		    "type": "archive",
+                    "path": "enroute-1.8.0.tar.xz"
```

Comment:
you should point to a source code release elswhere, rather than including it alongside the manifest such as this one:

https://github.com/Akaflieg-Freiburg/enroute/releases/tag/1.7.0

and example manifest layout:
https://github.com/flathub/org.filezillaproject.Filezilla/blob/master/org.filezillaproject.Filezilla.json#L92

you'll need to make sure to include a checksum to verify the source code like `sha256sum` or `sha512sum`.